tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
+| Signed Distance Field   | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| Global Illumination     |                          |                          |                          |
+| (SDFGI)                 |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Glow                    | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Adjustments             | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom post-processing  | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
+| with fullscreen quad    |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom post-processing  | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| with CompositorEffects  |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Antialiasing
+~~~~~~~~~~~~
+
+See :ref:`doc_3d_antialiasing` for more information.
+
++-------------------+--------------------------+--------------------------+--------------------------+
+| Feature           | Compatibility            | Mobile                   | Forward+                 |
++===================+==========================+==========================+==========================+
+| MSAA              | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| TAA               | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| FSR2              | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| FXAA              | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| SSAA              | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
```

Comment:
3D scaling in Compatibility is supported since 4.3, so SSAA is supported now (it just involves setting the render scale above `1.0`).

Reply:
This comment got resolved and the note was removed in the 3D antialising page, but it is still incorrect here.